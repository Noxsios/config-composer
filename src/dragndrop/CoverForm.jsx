import FilePickerYaml from "./FilePickerYaml";
import FilePickerMD from "./FilePickerMD";
import Grid from "@material-ui/core/Grid";

const CoverForm = () => {
  return (
    <div className="no-print">
      <Grid container spacing={1} direction="row" justify="center" alignItems="center" alignContent="center" wrap="nowrap">
        <Grid container spacing={1} direction="column" justify="center" alignItems="center" alignContent="center" wrap="nowrap">
          <FilePickerYaml />
        </Grid>
        <Grid container spacing={1} direction="column" justify="center" alignItems="center" alignContent="center" wrap="nowrap">
          <FilePickerMD />
        </Grid>
      </Grid>
    </div>
  );
};

export default CoverForm;
