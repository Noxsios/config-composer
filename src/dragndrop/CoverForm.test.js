import renderer from "react-test-renderer";
import CoverForm from "./CoverForm";

it("renders history modal skeleton", () => {
  const t = renderer.create(<CoverForm />).toJSON();
  expect(t).toMatchSnapshot();
});
