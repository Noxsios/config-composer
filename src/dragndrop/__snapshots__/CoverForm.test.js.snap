// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders history modal skeleton 1`] = `
<div
  className="no-print"
>
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column MuiGrid-align-items-xs-center MuiGrid-align-content-xs-center MuiGrid-justify-xs-center"
  >
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-align-items-xs-center MuiGrid-align-content-xs-center MuiGrid-justify-xs-center"
    >
      <div
        style={
          Object {
            "margin": "1rem",
            "minWidth": 300,
          }
        }
      >
        <div
          className="euiFilePicker euiFilePicker--large"
        >
          <div
            className="euiFilePicker__wrap"
          >
            <input
              aria-describedby="yaml-picker-filePicker__prompt"
              aria-label="YAML File Picker"
              className="euiFilePicker__input"
              id="yaml-picker"
              onChange={[Function]}
              onDragLeave={[Function]}
              onDragOver={[Function]}
              onDrop={[Function]}
              type="file"
            />
            <div
              className="euiFilePicker__prompt"
              id="yaml-picker-filePicker__prompt"
            >
              <svg
                aria-hidden={true}
                className="euiIcon euiIcon--large euiIcon-isLoading euiFilePicker__icon"
                focusable="false"
                height={16}
                role="img"
                style={null}
                viewBox="0 0 16 16"
                width={16}
                xmlns="http://www.w3.org/2000/svg"
              />
              <div
                className="euiFilePicker__promptText"
              >
                YAML
              </div>
            </div>
          </div>
        </div>
        <div
          className="Toastify"
        />
      </div>
      <div
        style={
          Object {
            "margin": "1rem",
            "minWidth": 300,
          }
        }
      >
        <div
          className="euiFilePicker euiFilePicker--large"
        >
          <div
            className="euiFilePicker__wrap"
          >
            <input
              aria-describedby="md-picker-filePicker__prompt"
              aria-label="Markdown File Picker"
              className="euiFilePicker__input"
              id="md-picker"
              onChange={[Function]}
              onDragLeave={[Function]}
              onDragOver={[Function]}
              onDrop={[Function]}
              type="file"
            />
            <div
              className="euiFilePicker__prompt"
              id="md-picker-filePicker__prompt"
            >
              <svg
                aria-hidden={true}
                className="euiIcon euiIcon--large euiIcon-isLoading euiFilePicker__icon"
                focusable="false"
                height={16}
                role="img"
                style={null}
                viewBox="0 0 16 16"
                width={16}
                xmlns="http://www.w3.org/2000/svg"
              />
              <div
                className="euiFilePicker__promptText"
              >
                Markdown
              </div>
            </div>
          </div>
        </div>
        <div
          className="Toastify"
        />
      </div>
    </div>
  </div>
</div>
`;
